python-crud-google-spreadsheet/README.md

```diff
@@ -33,7 +31,7 @@ $ export SPREADSHEET_ID={your-designated-spreadsheet-id}
 $ ./deploy.sh
 
 # Use the endpoints supplied to create, read, and modify data in the sheet using the binaris CLI or curl commands
-$ bn invoke create --data '{"values": [1, 2, 3, 4, 5]}'
+$ bn invoke create_endpoint --data '{"values": [1, 2, 3, 4, 5]}'
 $ curl -H X-Binaris-Api-Key:{your-binaris-api-key} 'https://run.binaris.com/v2/run/{your-account-id}/delete?key={row-number}'
 ```
```

Comment:
I think that the `create_endpoint` command should be followed by a `read_endpoint` command.